upgrade/upgrade-scripts/execute

```diff
@@ -380,6 +380,15 @@ dpkg-query -Wf '${Conffiles}\n' | awk '$3 == "obsolete" {print $1}' |
 			die "failed to reinstall package '$package'"
 	done || die "failed to remove obsolete package configuration files"
 
+#
+# Due to https://github.com/influxdata/telegraf/issues/14052, telegraf must be masked after
+# packages are upgraded. The telegraf package removes /etc/systemd/system/telegraf.service thus
+# reversing the `systemctl mask` operation performed before the packages are upgraded.
```

Comment:
Maybe add that once the upstream issue with telegraf has been fixed, this block can be removed?

Reply:
done